app/models/apply_application.rb

```diff
@@ -21,6 +22,14 @@ def application_attributes
     @application_attributes ||= parsed_application["attributes"]
   end
 
+  def mark_as_imported!
```

Comment:
Do we need these two methods? The enum gives us `ApplyApplication#imported!` and `ApplyApplication#non_importable_duplicate!`

Reply:
I tried this in the console earlier and it didn't work. I must have done something wrong because I had a gut feeling you could do this sort of stuff with enums.